scripts/xdebug.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# For running PHPStorm or IntelliJ, follow these steps:
+# 1. Go to Settings => Language & Frameworks => PHP => Servers
+# 2. Add you Vagrant server with the correct port.
+# 3. Check the pathmapping checkbox and set your root of the project to /vagrant and the www directory to /www/directory
+# 4. Apply and close settings
+# 5. Go to Run => Edit configurations
+# 6. Add a new PHP Web Application with the server you just added. The rest is trivial and free to fill.
+# 7. Aply and done!
+
+echo "Install Xdebug"
+yum install -y gcc gcc-c++ autoconf automake --enablerepo=epel --enablerepo=webtatic
```

Comment:
Please do, either way you need to remove it or include the webtatic/epel script.